src/Calendar.js

```diff
@@ -518,15 +518,7 @@ let Calendar = React.createClass({
     onNavigate(date, view)
 
     if (action === navigate.DATE)
-      this._viewNavigate(date)
-  },
-
-  _viewNavigate(nextDate) {
-    let { view, date, culture } = this.props;
-
-    if (dates.eq(date, nextDate, view, localizer.startOfWeek(culture))) {
```

Comment:
i'm working on something. the problem is conflating these two things in the first place, but i need to add some more props to fix it for good :P I'll have something up in a bit